site/docs/latest/development/licensing.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Third party dependencies and licensing
+---
+
+The bookkeeper project ships two binary distributions.
```

Comment:
better to call out LICENSE/NOTICE in source package vs LICENSE/NOTICE in binary package, when to update them accordingly.

Reply:
Added a section for source dependencies.